File: src/nomanssave/el.java
package nomanssave;

import java.io.File;
import javax.swing.Icon;
import javax.swing.filechooser.FileView;

class el extends FileView {
   // $FF: synthetic field
   final ej is;

   el(ej var1) {
      this.is = var1;
   }

   public Icon getIcon(File var1) {
      String var2;
      if (var1.isFile()) {
         var2 = var1.getName();
         if (var2.endsWith(".hg") && !var2.startsWith("mf_")) {
            return ej.as();
         } else {
            return var2.equals("containers.index") ? ej.au() : null;
         }
      } else {
         var2 = ej.a(this.is, var1);
         return var2 == null ? null : ej.aR();
      }
   }

   public String getName(File var1) {
      if (var1.isFile()) {
         return var1.getName();
      } else {
         String var2 = ej.a(this.is, var1);
         return var2 == null ? var1.getName() : "[" + var2 + "] " + var1.getName();
      }
   }
}

File: NMSSaveEditor/nomanssave/lower/el.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.IO;
using System.Linq;
using System.Text;
using System.Text.RegularExpressions;
using System.Windows.Forms;
using System.Globalization;

namespace NMSSaveEditor
{

public class el : FileView {
   // $FF: synthetic field
   ej @is;

   public el(ej var1) {
      this.@is = var1;
   }

   public Image getIcon(FileInfo var1) {
      string var2;
      if (var1.isFile()) {
         var2 = var1.Name;
         if (var2.EndsWith(".hg") && !var2.StartsWith("mf_")) {
            return ej.@as();
         } else {
            return var2.Equals("containers.index") ? ej.au() : null;
         }
      } else {
         var2 = ej.a(this.@is, var1);
         return var2 == null ? null : ej.aR();
      }
   }

   public string getName(FileInfo var1) {
      if (var1.isFile()) {
         return var1.Name;
      } else {
         string var2 = ej.a(this.@is, var1);
         return var2 == null ? var1.Name : "[" + var2 + "] " + var1.Name;
      }
   }
}

}
